aleph-client/check-runtime-metadata.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 
+set -e
+
 echo "This script must be run from the aleph-client directory."
+echo "Make sure you have aleph-node running in the background."
```

Comment:
You can check it by runnig `pidof aleph-node`, but it's not strictly necessary.

Reply:
Done